packages/competence/test/README.md

```
@@ -0,0 +1,165 @@
+# Competence Package Test Suite
+
+This directory contains comprehensive tests for the ti-engine competence package.
+
+## Test Files
+
+### 1. json-config-validation.test.js
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)